ext_mongodb.php

```diff
@@ -853,21 +1100,51 @@ public function __debugInfo() : array
 
 final class UTCDateTime implements Type, \Serializable
 {
-	use DenySerialization;
-
 	private int $milliseconds;
 
+	static private function checkArray( array $state )
+	{
+		if (
+			!array_key_exists( 'milliseconds', $state ) ||
+			! ( is_int( $state['milliseconds'] ) || is_string( $state['milliseconds'] ) )
+		) {
+			throw new \MongoDB\Driver\Exception\InvalidArgumentException( "MongoDB\BSON\UTCDateTime initialization requires \"milliseconds\" integer or numeric string field" );
+		}
+	}
+
+	public function serialize() : string
+	{
+		return serialize( [
+			'milliseconds' => (string) $this->milliseconds,
```

Comment:
PHPC uses integers on 64-bit platforms. Shouldn't you also do that here?

I currently support string values on initialization, which handles the case of unpacking 32-bit serialization data on a 64-bit system. Did you perhaps have [PHPC-726](https://jira.mongodb.org/browse/PHPC-726) in mind when writing this?